cli/cmd/compliance_aws.go

```diff
@@ -243,6 +245,135 @@ To show recommendation details and affected resources for a recommendation id:
 			return nil
 		},
 	}
+
+	// complianceAwsDisableReportCmd represents the disable-report sub-command inside the aws command
+	// experimental feature
+	complianceAwsDisableReportCmd = &cobra.Command{
+		Use:     "disable-report <report_type>",
+		Aliases: []string{"disable"},
+		Hidden:  true,
+		PreRunE: func(_ *cobra.Command, args []string) error {
+			switch args[0] {
+			case "CIS":
+				args[0] = fmt.Sprintf("AWS_%s_S3", args[0])
+				return nil
+			case "AWS_CIS_S3":
+				return nil
+			default:
+				return errors.New("CIS is the only supported report type")
+			}
+		},
+		Args: cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+
+			schema, err := fetchCachedAwsComplianceReportSchema(args[0])
+			if err != nil {
+				return errors.Wrap(err, "unable to get aws compliance report schema")
+			}
+
+			// set state of all recommendations in this report to disabled
+			patchReq := api.NewRecommendationV1State(schema, false)
+			response, err := cli.LwApi.Recommendations.Aws.Patch(patchReq)
+			if err != nil {
+				return errors.Wrap(err, "unable to patch aws recommendations")
+			}
+
+			var cacheKey = fmt.Sprintf("compliance/aws/schema/%s", args[0])
+			cli.WriteAssetToCache(cacheKey, time.Now().Add(time.Minute*30), response.RecommendationList())
+			cli.OutputHuman(fmt.Sprintf("All recommendations for report %s have been disabled\n", args[0]))
```

Comment:
You don't need to concatenate the string, this function already works similar to `fmt.Sprintf()`

Fix all other statements like this one, please.
```suggestion
			cli.OutputHuman("All recommendations for report %s have been disabled\n", args[0])
```

Reply:
👍